docs/adr/0006-treat-rollouts-differently.md

```diff
@@ -0,0 +1,31 @@
+# Opting out of studies should not affect rollouts
+
+> Status: Open
+
+> Author: Travis Long
+
+> Deciders: Nimbus team, Trust & Legal
+
+> Date: H1 2022
+
+## Context and Problem Statement
+
+Rollouts are a feature of Nimbus and are distinct from experiments in that they are not measurement instruments but instead represent off-train configuration updates.
+
+Rollouts can serve many purposes:
+- Launching a winning branch of an experiment faster than the trains.
+- Launching a configuration to non-experiment users during an experiment after a short period of verification
+- Configuring different settings for a feature for different audiences remotely
+- A "kill switch" if you want to launch a feature but then turn it off if something goes wrong.
+
```

Comment:
Self-review: note to add information about the messaging requirement for being able to send messages to users who have opted out of experiments.